orchestra/tests/helpers/__init__.py

```diff
@@ -73,3 +74,17 @@ def assertModelInstanceExists(self,
     def assertModelInstanceNotExists(self, model_class, lookup_attributes):
         with self.assertRaises(model_class.DoesNotExist):
             model_class.objects.get(**lookup_attributes)
+
+
+class OrchestraTestCase(OrchestraTestHelpersMixin, TestCase):
+    # NOTE(lydia): Mixin should go before TestCase because when mixin calls
```

Comment:
what I find strange is that do people create mixins which must be inherited with another class?